File: push/src/main/java/top/trumeet/mipushframework/wizard/FinishedPhonyPermissionInfo.java
package top.trumeet.mipushframework.wizard;

import android.content.Context;

import androidx.annotation.NonNull;

import com.xiaomi.xmsf.R;

public class FinishedPhonyPermissionInfo extends DisplayOnlyPhonyPermissionInfo {

    public FinishedPhonyPermissionInfo(Context context) {
        super(context);
    }

    @NonNull
    @Override
    public String getPermissionTitle() {
        return context.getString(R.string.app_name);
    }

    @NonNull
    @Override
    public String getPermissionDescription() {
        return context.getString(R.string.wizard_descr_finish);
    }

}

File: push/src/main/java/top/trumeet/mipushframework/wizard/FinishedPhonyPermissionInfo.kt
package top.trumeet.mipushframework.wizard

import android.content.Context
import com.xiaomi.xmsf.R

class FinishedPhonyPermissionInfo(context: Context) : DisplayOnlyPhonyPermissionInfo(context) {
    override val permissionTitle: String
        get() = context.getString(R.string.app_name)

    override val permissionDescription: String
        get() = context.getString(R.string.wizard_descr_finish)
}
